code/__defines/subsystem-defines.dm

```diff
@@ -118,7 +118,7 @@
 // - SSjobs --
 // departments
 #define DEPARTMENT_COMMAND "Command"
-#define DEPARTMENT_COMMAND_SUPPORT "Command Support"
+#define DEPARTMENT_COMMAND_SUPPORT "Support"
```

Comment:
No, keep it Command Support

Reply:
I'm changing it to command support because having two words for a department is too unwieldly and somewhat confusing, I feel.
Also already done by Bay so there's at least some precedent.